aptos-move/block-executor/src/executor.rs

```diff
@@ -978,13 +1085,24 @@ where
     }
 
     fn apply_output_sequential(
-        unsync_map: &UnsyncMap<T::Key, T::Value, X, T::Identifier>,
+        unsync_map: &UnsyncMap<T::Key, T::Tag, T::Value, X, T::Identifier>,
         output: &E::Output,
-    ) {
+    ) -> ::std::result::Result<(), PanicOr<IntentionalFallbackToSequential>> {
         for (key, (write_op, layout)) in output.resource_write_set().into_iter() {
             unsync_map.write(key, write_op, layout);
         }
 
+        // TODO[agg_v2](fix): provide layouts
+        for (group_key, _, group_ops) in output.resource_group_write_set().into_iter() {
+            for (value_tag, group_op) in group_ops.into_iter() {
+                unsync_map
+                    .insert_group_op(&group_key, value_tag, group_op)
+                    .map_err(|_| {
+                        PanicOr::Or(IntentionalFallbackToSequential::ResourceGroupError)
```

Comment:
But we are already in sequential mode?

Reply:
+1

Reply:
@igor-aptos should we also mark these as agg_v2 todos? i.e. now we have some invariants that maybe with thorough hardening and testing we can fill confident to assert in sequential execution (because there is no more fallback from there), but we also have the serialization error that we still have to find a good solution for.

Reply:
FallbackToSequential during sequential execution is basically CodeInvariantError, will make things more explicit separately